signac/__main__.py

```diff
@@ -718,6 +697,27 @@ def main_config_show(args):
             print(line)
 
 
+def _verify_config(cfg, preserve_errors=True):
```

Comment:
Aside from the name change from `verify_config` to `_verify_config`, this is a pure move, so that this function is next to `main_config_verify`.